hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMClientProtocolServer.java

```diff
@@ -560,6 +563,13 @@ public boolean getReplicationManagerStatus() {
     return scm.getReplicationManager().isRunning();
   }
 
+  @Override
+  public List<String> getScmRatisStatus() throws IOException {
+    return scm.getScmHAManager()
+        .getRatisServer().getRaftPeers()
+        .stream().map(peer -> peer.getAddress()).collect(Collectors.toList());
```

Comment:
One thing I am not sure is what should be fetched from Ratis peers. Should I also include which one is leader and which one is follower?

Reply:
Could potentially throw NPE

Reply:
I can see this method be wrapper in ScmHAManager to show a list of scm hosts.

Reply:
Got it. Will move this functionality to ScmHAManager.